src/gazebo_geotagged_images_plugin.cpp

```diff
@@ -736,7 +736,7 @@ void GeotaggedImagesPlugin::_send_capture_status(struct sockaddr* srcaddr)
         interval,                               // image interval
         0,                                      // recording time in ms
         available_mib,                          // available storage capacity
-        0);                                     // total number of images
+        _imageCounter);                                     // total number of images
```

Comment:
Can you just fix the indentation? Otherwise, good to go!

Reply:
Oh I hate stuff like that which needs to align. It never does, fine :smile:

Reply:
At least without tabs there is one truth.